src/content/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring.mdx

```diff
@@ -1,325 +1,205 @@
-        * [Categories and labels for monitors](/docs/apis/synthetics-rest-api/monitor-examples/use-synthetics-label-apis)
-        * [Alert notifications](/docs/apis/synthetics-rest-api/alert-examples/manage-synthetics-alert-notifications-rest-api)
-      </td>
-    </tr>
-  </tbody>
-</table>
+<figcaption>
+Go to **[one.newrelic.com](https://one.newrelic.com) > Add data** > search for **Synthetic monitoring** > choose **Simple browser monitor**. 
+</figcaption>
+</Side>
+</SideBySide>
+
+</Step>
+
+<Step>
+
+## Create a scripted browser monitor [#browser]
+
+Ping monitors won't execute JavaScript because they aren't a full browser, just an HTTP client. Simple browser monitors, while using a Google Chrome instance, don't perform complex checks on your web page. 
+
+Scripted browser monitors give you full Javascript functionality so your monitors can check the availability of the individual elements in your web app. 
+
+<SideBySide>
+    <Side>
+    ### Go to a URL [#advanced-url]
+    The beginning of all scripts specifies the URL the monitor should check.
+
+    </Side>
+    <Side>
+
+```js
+//Visit http://example.com/
+$webDriver.get("http://example.com/");  
+```
+
+</Side>
+</SideBySide> 
+
+
+
+<SideBySide>
+    <Side>
+    ### Interact with a page element [#advanced-nav]
+
+Once your monitor goes to a URL, it scans for then interacts with the page's available elements. This script: 
+    1. Finds the **About** page via link text, then clicks the link: `$webDriver.findElement($driver.By.linkText("About")).click()` 
+    2. Searches for the partial string `Home` to find the the **Example site Home** buton, then clicks the link: `$webDriver.findElement($driver.By.partialLinkText("Home")).click();});`
+    </Side>
+    <Side>
+
+```js
+$webDriver.get("http://example.com/").then(function(){
+//Find a link with display text that matches `About`, then click that link.
+    return $webDriver.findElement($driver.By.linkText("About")).click();
+}).then(function(){
+    return $webDriver.findElement($driver.By.partialLinkText("Home")).click();
+});
+```
+</Side>
+</SideBySide> 
+
+<SideBySide>
+    <Side>
+### Search a website [#advanced-search]
+
+Beyond text sweeps, your scripted browser monitor can locate elements via their `class`, `id`, `name`, or XPath. This script:
+
+1. Goes to your site.
+2. Locates a search box via its XPath, then types `relic`.
+3. Locates a submit button via its XPath, then clicks it to submit the search.
+
+    </Side>
+    <Side>
+```js
+$webDriver.get("http://example.com/static/companyBlog.jsp").then(function(){
+//Find the search field by specifying its `id`, then enter `relic`.
+    return $webDriver.findElement($driver.By.xpath("//h4[text()='Blog Search']/following-sibling::div/input")).sendKeys("relic");
```

Comment:
Should use $selenium instead of $driver.

Also, this is an advanced xpath identifier. We may be able to find a better example that doesn't need to be this complex.